mypy/build.py

```diff
@@ -281,6 +281,7 @@ def default_lib_path(data_dir: str, pyversion: Tuple[int, int]) -> List[str]:
                         ('data_mtime', float),  # mtime of data_json
                         ('data_json', str),  # path of <id>.data.json
                         ('suppressed', List[str]),  # dependencies that weren't imported
+                        ('child_modules', Set[str]),  # all submodules of the given module
```

Comment:
I think this should be a list, since it closely reflects the JSON data model. You can turn it into a set right when incorporating it into the State object.